src/ResourceManager/NotificationHubs/Commands.NotificationHubs/Commands/Namespace/SetAzureNotificationHubsNamespace.cs

```diff
@@ -78,7 +84,7 @@ public override void ExecuteCmdlet()
                 () =>
                 {
                     // Update a namespace 
-                    var nsAttribute = Client.UpdateNamespace(ResourceGroup, Namespace, Location, State, Critical, ConvertTagsToDictionary(Tags));
+                    var nsAttribute = Client.UpdateNamespace(ResourceGroup, Namespace, Location, State, Critical, ConvertTagsToDictionary(Tags), skuTier);
```

Comment:
@amolr does `SkuTier` need to have a default value here like it does in `New-AzureNotificationHubsNamespace`? For example, if someone passes in an empty string for `SkuTier`, does it need to be set to "free"?